src/translation-service.js

```diff
@@ -0,0 +1,72 @@
+/**
+ * @license Copyright (c) 2003-2016, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+/**
+ * @module utils/translation-service
+ */
+
+const translations = {};
+
+/**
+ * Merges package translations to existing ones.
+ * These translations can be used later with {@link module:utils/translations-service~translate translate}
+ *
+ *		add( 'pl', {
+ *			'OK': 'OK',
+ *			'Cancel [context: reject]': 'Anuluj'
+ *		} );OK
+ *
+ * @param {String} lang
+ * @param {Object.<String, Object>} packageDictionary
+ * @returns undefined
+ */
+export function add( lang, packageDictionary ) {
+	if ( !( lang in translations ) ) {
+		translations[ lang ] = {};
+	}
+
+	const dictionary = translations[ lang ];
+
+	for ( const translationKey in packageDictionary ) {
+		const translation = packageDictionary[ translationKey ];
+		dictionary[ translationKey ] = translation;
+	}
+}
+
+/**
+ * Translates string if the translation of the string was previously added using {@link module:utils/translations-service~add add}.
+ * Otherwise returns original (English) sentence.
+ *
+ *		translate( 'pl', 'Cancel [context: reject]' );
+ *
+ * @param {String} lang Translation language.
```

Comment:
Incorrect description. It's not "translation's language" (it doesn't make any sense). It's more like "Target language." or "Language which dictionary will be used.".